src/app/layout.tsx

```diff
@@ -1,50 +1,35 @@
 "use client";
-import Header from "@/component/Header";
-import SideNav from "@/component/Sidenav";
+import Header from "@/components/Header";
+import SideNav from "@/components/Sidenav";
 import { Box } from "@mui/material";
 import { AppRouterCacheProvider } from "@mui/material-nextjs/v14-appRouter";
 import { ThemeProvider } from "@mui/material/styles";
-import { Roboto } from "next/font/google";
-import { useState } from "react";
+import { useStore } from "@/store/useStore";
 import "./globals.css";
 import theme from "./theme";
 
-const roboto = Roboto({
-  weight: ["300", "400", "500", "700"],
-  subsets: ["latin"],
-  display: "swap",
-  variable: "--font-roboto",
-});
-
 export default function RootLayout({
   children,
 }: Readonly<{ children: React.ReactNode }>) {
-  const [SideNavOpen, setSideNavOpen] = useState(false);
+  const { sideNavOpen } = useStore();
```

Comment:
Let's use the zustand stores sparsely. As a rule of thumb, when the components sharing the state are grouped and hierarchically not too distant, we should use useState and props drilling. Like in this case with Header and SideNav, they are close enough, using useState is fine.

Reply:
Done